phys/module_sf_noahmplsm.F

```diff
@@ -8040,6 +8043,13 @@ SUBROUTINE SPRINKLER_IRRIGATION(parameters,NSOIL,DT,SH2O,SMC,SICE,& !in
     IREVPLOS = IRSIRATE*IRRLOSS*(1./100.) 
     IRSIRATE = IRSIRATE-IREVPLOS
   END SUBROUTINE SPRINKLER_IRRIGATION
+
+    logical function isnan_lsm(arg1)
+        real,intent(in) :: arg1
+       isnan_lsm = (arg1 .ne. arg1)
+       return
+    end function isnan_lsm
+
```

Comment:
This "roll your own" ISNAN() function is the technique used in other physics packages (P3 MP, TEMF BL). I am entirely OK with this solution.

Reply:
Hi @davegill, OK thanks. We will go with our solution here by defining the isnan_lsm function as done in other physics packages.